closed/make/Docs.gmk

```diff
@@ -1,5 +1,5 @@
 # ===========================================================================
-# (c) Copyright IBM Corp. 2018, 2018 All Rights Reserved
+# (c) Copyright IBM Corp. 2018, 2019 All Rights Reserved
```

Comment:
Is this file derived from OpenJDK content?

Reply:
No.  The only knowledge derived from OpenJDK are are the names of the macros whose values are overridden.